src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -0,0 +1,288 @@
+﻿using System;
+using System.Linq;
+using Windows.UI.Input;
+using Windows.UI.Xaml.Input;
+using Uno.Extensions;
+using Uno.Logging;
+
+namespace Windows.UI.Xaml
+{
+	/*
+	 *	This partial file
+	 *		- Ensures to raise the right PointerXXX events sequences
+	 *		- Handles the gestures events registration, and adjusts the configuration of the GestureRecognizer accordingly
+	 *		- Forwards the pointers events to the gesture recognizer and then raise back the recognized gesture events
+	 *	
+	 *	The API exposed by this file to its native parts is:
+	 *		partial void InitializePointersPartial();
+	 *	
+	 *	The native components are responsible to subscribe to the native touches events,
+	 *	create the corresponding PointerEventArgs and then invoke one of the following methods:
+	 *		private bool RaiseNativelyBubbledDown(PointerRoutedEventArgs args);
+	 *		private bool RaiseNativelyBubbledMove(PointerRoutedEventArgs args);
+	 *		private bool RaiseNativelyBubbledUp(PointerRoutedEventArgs args);
+	 *		private bool RaiseNativelyBubbledLost(PointerRoutedEventArgs args);
+	 *
+	 */
+
+	partial class UIElement
+	{
+		#region ManipulationMode (DP)
+		public static readonly DependencyProperty ManipulationModeProperty = DependencyProperty.Register(
+			"ManipulationMode",
+			typeof(ManipulationModes),
+			typeof(UIElement),
+			new FrameworkPropertyMetadata(ManipulationModes.System, FrameworkPropertyMetadataOptions.None, OnManipulationModeChanged));
+
+		private static void OnManipulationModeChanged(DependencyObject snd, DependencyPropertyChangedEventArgs args)
+		{
+			if (snd is UIElement elt)
+			{
+				elt.PrepareParentTouchesManagers((ManipulationModes)args.NewValue);
+			}
+		}
+
+		public ManipulationModes ManipulationMode
+		{
+			get => (ManipulationModes)this.GetValue(ManipulationModeProperty);
+			set => this.SetValue(ManipulationModeProperty, value);
+		}
+		#endregion
+
+		private /* readonly */ Lazy<GestureRecognizer> _gestures;
```

Comment:
commented code

Reply:
It should be `readonly`, but I cannot actually define it due to partial classes. I prefered to let it here for information.

Reply:
Mention it in the comments.